src/Compilers/Core/Portable/Operations/IDynamicObjectCreationExpression.cs

```diff
@@ -12,6 +12,11 @@ namespace Microsoft.CodeAnalysis.Semantics
     public interface IDynamicObjectCreationExpression : IHasDynamicArgumentsExpression
     {
         /// <summary>
+        /// Name of the dynamically invoked member.
+        /// </summary>
+        string MemberName { get; }
```

Comment:
>string MemberName { get; } [](start = 8, length = 26)

I am not sure if this property belongs in IDynamicObjectCreationExpression. Isn't the member is always an instance constructor, which has predefined name?

Reply:
The doc comment is actually misleading, it should be name of the dynamically created type, and the field name should be "Name".

Reply:
Hmm, users can derive this name from syntax or the child dynamic member reference expression. Let me remove this property.

Reply:
>  users can derive this name from syntax or the child dynamic member reference expression.

I assume the type symbol is available as the Type property of the node itself. Correct?